Code/ThirdParty/Jolt/Physics/Body/BodyAccess.h

```diff
@@ -8,7 +8,18 @@
 
 JPH_NAMESPACE_BEGIN
 
-class BodyAccess
+/// EZ Modification start
+/// This is a workaround for clang requiring dll export of static thread_local members.
+#ifdef __clang__
+  #undef EZ_CLANG_JPH_EXPORT
```

Comment:
Let's put this next to the definition of `JPH_EXPORT` so we don't have to repeat it multiple times.

Reply:
Also I'm 90% sure this requirement is linux only. Your fix however also kicks in when compiling with clang on windows. My assumption would be, that this now breaks the debug build on windows when compiling with clang.

Reply:
I'm not thrilled to have an ez specific change in this library, that I can't merge back. Could you maybe rename this and clean it up, so that I can submit this to jolt?

Reply:
I can try. Need to look into how Jolt defines their stuff. THe problem seems to be that:
1. Windows: "data with thread storage duration may not have dll interface"
2. Linux clang: exact oposite of windows.
3. Linux gcc: does not care one way or the other.

Reply:
You can probably keep this extra define, but rename it to something like JPH_EXPORT_CLANG. And change the comments to not mention EZ.

Reply:
I've pushed new changes, pleaes have a look!